lightning/restore/restore.go

```diff
@@ -474,6 +472,66 @@ func (rc *RestoreController) runPeriodicActions(ctx context.Context, stop <-chan
 	}
 }
 
+type gcLifeTimeManager struct {
+	runningJobsLock sync.Mutex
+	runningJobs     int
+	oriGCLifeTime   string
+}
+
+func newManager() *gcLifeTimeManager {
```

Comment:
Oh also please rename this method to something more specific, since it is seen by the whole package 😅. 

```suggestion
func newGCLifeTimeManager() *gcLifeTimeManager {
```